examples/src/Viewer.tsx

```diff
@@ -412,6 +416,7 @@ class Viewer extends React.Component<InputParams, ViewerState> {
         // if (!fileName.includes(".simularium")) {
         //     return new
         // }
```

Comment:
I know it's not part of this code, but could this commented out code be removed ?

Reply:
sure :)